### DISCUSSION TITLE
Mindmaps for {episoap} pipelines

### DISCUSSION THREAD

**Discussion by CarmenTamayo** [WHO-Collaboratory/collaboratory-epipipeline-community#7] [General]
Summary of discussions

**Comment by CarmenTamayo** [comment]
Scope of {episoap}: Potential overlap with {sitrep} by AppliedEpi. 
From our discussions, {episoap} would focus on middle and late tasks, as defined by [Epiverse-TRACE pipelines](https://github.com/epiverse-trace/epiverse-trace.github.io/pull/88).
{episoap} also highlights alternatives for specific analysis paths, offering different options depending on user needs, e.g., different packages to use.

The package {sitrep} by AppliedEpi offers templates to produce reports, however it seems like, at least for now, they are focusing on early outbreak tasks, e.g., epi curves, spatial distribution of cases, etc.

**Comment by CarmenTamayo** [comment]
Tone of {episoap}: at the moment, there is a mix of text targeted to the final reader (e.g., policy maker or senior manager), as well as text directed to the person who's writing the report and using the pipeline, e.g., in the [transmissibility pipeline](https://github.com/epiverse-trace/episoap/tree/main/inst/rmarkdown/templates/transmissibility).

- Interpretation of outputs in figures
- Automated testing of interoperability of packages

**Comment by Bisaloo** [comment]
episoap can also be used as a vector for the "integration testing" proposed during the idea identification phase.

The pipelines can run on a regular basis in the cloud and breakages would signal new incompatibility & breaking changes.

**Comment by calthaus** [comment]
Notes on the workflow with pipelines, dimensions, tags, etc.: https://docs.google.com/document/d/13rX4Qkds_ZztFLP9vakjMFwK3QojpIypVwKxsRITkxA/edit?usp=sharing

Table of dimensions/tags: https://docs.google.com/spreadsheets/d/1iliTbQIq2-z04fD8wN8bmXvT0RXfC3-A_3cZoa6CFUI/edit?usp=sharing

**Comment by sarahollis** [comment]
My comments: 
seems really sensible and valuable addition for developer community and R users alike - given there are currently a lot of different ways to approach certain analyses and it could be very helpful to have streamlined out-of-the-box solutions for them.

it would take a good amount of sensitization for the prospective tagging of new scripts/packages/analyses in the future - but agree would be important for the repository to grow over time and add new resources/contributions. what what would the review process be like? (you would want to maintain degree of quality standards etc) and who would actually sift through all of that? could quickly become a big task.

**Comment by yuriei** [comment]
Great job to the team! Looks great with a comprehensive summary. While the number of criteria looking at this way seems a bit overwhelming, but I understand the needs and challenges there. I know this would be a long-term objective but having any interface for the user / more easy to follow documentation on each option (ie. how do you define the phase of your outbreak? When should I choose projective modelling vs forecasting? etc) would also be helpful.
